protected/humhub/config/common.php

```diff
@@ -110,6 +110,10 @@
         'cache' => [
             'class' => \yii\caching\DummyCache::class,
         ],
+        'runtimeCache' => [
+            'class' => \yii\caching\ArrayCache::class,
```

Comment:
This could possibly lead to memory overruns for console jobs such as integrity check or sending mail summary.

Maybe we should set this to Dummy Cache on Console?

Reply:
Got it